src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -51,6 +51,53 @@ internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProvider
 
         private static readonly ParseOptions CSharp7_1 = CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7_1);
 
```

Comment:
The IDE changes should be omitted from this

Reply:
I'll keep the added test, since it's the original scenario that prompted this change.